lib/common.sh

```diff
@@ -340,13 +341,14 @@ function init_minikube() {
 
 
 #
-# Create the CRs for v1alpha2 deployments
+# Create the CRs for v1alpha2 or v1alpha3 deployments
 #
 # Inputs:
 # - machine type (controlplane or workers)
 #
-make_v1alpha2_machine() {
+make_v1alphaX_machine() {
     MACHINE_TYPE=$1
+    ALPHA_VERSION=$2
```

Comment:
you could take the CR path as a parameter instead

Reply:
v1alpha2 and v1alpha3 used. Descriptive and not too long